File: src/library/scala/runtime/java8/JFunction0$mcC$sp.java

/*
 * Copyright (C) 2012-2015 Typesafe Inc. <http://www.typesafe.com>
 */

package scala.runtime.java8;

@FunctionalInterface
public interface JFunction0$mcC$sp extends scala.Function0, java.io.Serializable {
    char apply$mcC$sp();

    default Object apply() { return scala.runtime.BoxesRunTime.boxToCharacter(apply$mcC$sp()); }
}

File: src/library/scala/runtime/java8/JFunction0$mcC$sp.scala
/*
 * Copyright (C) 2012-2015 Typesafe Inc. <http://www.typesafe.com>
 */
package scala.runtime.java8

@FunctionalInterface trait JFunction0$mcC$sp extends Function0[Any] with Serializable {
  def apply$mcC$sp: Char
  override def apply: Any = scala.runtime.BoxesRunTime.boxToCharacter(apply$mcC$sp)
}